src/app.module.ts

```diff
@@ -71,6 +72,7 @@ export class AppModule implements NestModule {
         ContractsModule,
         DataDecodedModule,
         DelegatesModule,
```

Comment:
Nit: we should add a TODO comment to delete the legacy route, and remove `V2` accordingly.

Reply:
I completely agree. Added in [5e84006](https://github.com/safe-global/safe-client-gateway/pull/1501/commits/5e840060faeb683548aa6d7fb706ea76a048844e), thanks!